whisper/whisperv6/whisper.go

```diff
@@ -176,14 +181,41 @@ func (w *Whisper) SetMaxMessageSize(size uint32) error {
 }
 
 // SetMinimumPoW sets the minimal PoW required by this node
-func (w *Whisper) SetMinimumPoW(val float64) error {
+func (w *Whisper) SetMinimumPoW(val float64, testMode bool) error {
```

Comment:
It would make more sense to Mock `SetMinimumPoW` during your tests instead of explicitly changing the function interface to specify if this is a test or not. Many things can slip through the cracks this way.

Reply:
do you mean, i should create another function, e.g. SetMinimumPowTest?

Reply:
yes, have a `Whisper` as an `interface` that implements `SetMinimumPoW`. Then, in tests, implement the `Whisper` interface as a `WhisperMock` structure and implement a different `SetMinimumPoW` for this. However, as I'm saying below, I think this is overkill when you can simply use the same function in both cases.